### PR TITLE
Upgrade to Spring boot 2.7.6

### PR DESCRIPTION
**Pull request by swaranga-netflix**

### CODE REVIEW

File: metacat-main/build.gradle [hunk 1]
```diff
@@ -76,9 +77,10 @@ dependencies {
     runtime(project(":metacat-connector-druid"))
 
     runtime("io.springfox:springfox-swagger-ui")
-    runtime("org.webjars:hal-browser")
+    runtime("org.webjars:hal-browser:3325375")
```
Comment (rsyncv) [inline metacat-main/build.gradle:80]: Curious what the reason for this pinning is

File: metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy [hunk 2]
```diff
@@ -123,6 +123,16 @@ class MetacatSmokeSpec extends Specification {
             if (metadata != null) {
                 newTable.getMetadata() == null? newTable.setMetadata(metadata): newTable.getMetadata().putAll(metadata)
             }
+
+            if (tableName == 's3-mysql-db') {
```
Comment (rsyncv) [inline metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy:127]: Would it make sense to just set the owner for all tables created

File: metacat-main/build.gradle (hunk 1 above)
Comment (swaranga-netflix) [reply metacat-main/build.gradle:80]: The new Netflix BOm does not have recommendations for this artifact, so we have to specify a specific version

File: metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy (hunk 2 above)
Comment (swaranga-netflix) [reply metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy:127]: It is mandated only for the S3 connector, but let me try and see what breaks.

**Review by rsyncv — APPROVED** [review]

**Pull request merged**